.github/workflows/testing-and-deployment.yml

```diff
@@ -294,7 +294,8 @@ jobs:
         run: pip install . --no-deps
 
       - name: Install test dependencies
-        run: pip install vtk -r requirements_test.txt
+        # https://github.com/pyvista/pyvista/pull/6458
+        run: pip install vtk --only-binary "matplotlib" -r requirements_test.txt
```

Comment:
Does `--only-binary` work like this, in the middle? These switches usually apply globally to the whole install incantation.

Reply:
Yes. We can specify the package individually.
https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-no-binary
>  or one or more package names with commas between them (no colons).